README.rst

```diff
@@ -16,8 +16,4 @@ common helpers used by all Google API clients. For more information, see the
 
 Supported Python Versions
 -------------------------
-Python >= 3.5
-
-Deprecated Python Versions
---------------------------
-Python == 2.7. Python 2.7 support will be removed on January 1, 2020.
```

Comment:
IIRC this section should be retained, if only to explain what the last Python 2.7 compatible version is.

BigQuery example: https://googleapis.dev/python/bigquery/latest/index.html#unsupported-python-versions
Pub/Sub docs example: https://googleapis.dev/python/pubsub/latest/index.html#deprecated-python-versions

Reply:
fe7d454